buildSrc/src/main/java/org/elasticsearch/gradle/util/Util.java

```diff
@@ -7,7 +7,7 @@
  * not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
+ *         http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
accidental indentation?

Reply:
Yeah, this is actually messed up in `ide.gradle`. I've fixed that as well.